Tests/AppcuesKitTests/Traits/TraitComposerTests.swift

```diff
@@ -56,9 +56,12 @@ class TraitComposerTests: XCTestCase {
             ],
             redirectURL: nil,
             nextContentID: nil)
+        let stepIndex = Experience.StepIndex(group: 0, item: 0)
+        let package = try traitComposer.package(experience: ExperienceData(experience, trigger: .showCall), stepIndex: stepIndex)
 
         // Act
-        _ = try traitComposer.package(experience: ExperienceData(experience, trigger: .showCall), stepIndex: Experience.StepIndex(group: 0, item: 0))
+        // simulate the presentation, this updater called from ExperienceStateMachine.SideEffect.executePresentContainer
+        try package.stepDecoratingTraitUpdater(stepIndex.item, nil)
```

Comment:
fyi @mmaatttt on these test changes to match - I had missed these earlier. I believe the primary intent of these is to  test trait propagation, and this update will preserve the same underlying test - though it's now triggered in a different fashion.

Reply:
Yep, perfect